File: 231109/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int s, age;

    cin >> s >> age;

    if(s == 1) {
        if(age >= 19) {
            cout << "WOMAN";
        }
        else {
            cout << "GIRL";
        }
    }
    else {
        if(age >= 19) {
            cout << "MAN";
        }
        else {
            cout << "BOY";
        }
    }
    
    return 0;
}

File: 250211/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt(), b = sc.nextInt();

        if(a == 1) {
            if(b >= 19) {
                System.out.print("WOMAN");
            }
            else {
                System.out.print("GIRL");
            }
        }
        else {
            if(b >= 19) {
                System.out.print("MAN");
            }
            else {
                System.out.print("BOY");
            }
        }
    }
}